TelCo.ColorCoder/Program.cs

```diff
@@ -1,147 +1,49 @@
-﻿using System;
+using System;
 using System.Diagnostics;
 using System.Drawing;
+using static TelCo.ColorCoder.ColorMap;
 
 namespace TelCo.ColorCoder
 {
-    /// <summary>
-    /// The 25-pair color code, originally known as even-count color code, 
-    /// is a color code used to identify individual conductors in twisted-pair 
-    /// wiring for telecommunications.
-    /// This class provides the color coding and 
-    /// mapping of pair number to color and color to pair number.
-    /// </summary>
     class Program
```

Comment:
Duplication code in  Console.WriteLine (......) .....violation of DRY principle

Reply:
Created separate functions to print the result of the functions in order to avoid the duplication code